### DISCUSSION TITLE
i bought two licenses as a mistake but i need two just read it until the end

### DISCUSSION THREAD

**Discussion by ItxFilas** [aclap-dev/video-downloadhelper#1318] [License & Purchase requests]
### What's your issue?

Hi i have bought a licence for chrome two times but i really need it only for one, i can send you recipt can you give me firefix license? i will send reciepts on email or idk just not in public, contact me asap sekylogins@protonmail.com

### The 10 first digits (only 10) of your license number or email address

84fd8518675

### I confirm that I did not not paste my full license number

- [X] I confirm

**Comment by jcfrog** [comment]
Hi,
I sent you an email.

**Reply by ItxFilas** [reply]
I didn't receive

**Comment by jcfrog** [comment]
Do you mean you purchased another license and did not receive it?
jerome

**Comment by ItxFilas** [comment]
Nah I mean I purchased wrong one, I bought chrome and I need for Firefox
and chrome (bought 2 chromes) I didn't know it's separate

**Reply by jcfrog** [reply]
I can only find one Chrome premium from 2021.
We can't swap licenses. 
Chrome and Firefox premium are separate products with different prices and features.
